src/app/about/page.js

```diff
@@ -2,13 +2,15 @@ import WhoWeAre from "@/components/about/WhoWeAre";
 import React from "react";
```

Comment:
Only file that should be edited for this issue is the `Header.jsx`
Please revert your changes from this file